promoter/database.go

```diff
@@ -60,6 +66,10 @@ var (
 	// updateMaxBatchSize is the max number of addresses we send to skyd
 	// within a single API request.
 	updateMaxBatchSize = minUnusedAddresses
+
+	// configIDConversionRate is the ID of the currency conversion rate in
+	// the config collection.
+	configIDConversionRate = primitive.ObjectID{1}
```

Comment:
Wouldn't it be more convenient to use keys and values instead of ids? This is what I did in `pinner` when I needed global configuration: https://github.com/SkynetLabs/pinner/blob/main/conf/configuration.go#L45
Your approach works as well, the only difference is how it reads when you look at the DB in Robo3.